local-cluster/src/local_cluster.rs

```diff
@@ -668,7 +679,8 @@ impl LocalCluster {
                 },
                 amount,
                 vote_instruction::CreateVoteAccountConfig {
-                    space: vote_state::VoteStateVersions::vote_state_size_of(true) as u64,
```

Comment:
this was fine before because all the multi node local cluster tests used ClusterType::Development which turns on all the flags.